mungegithub/mungers/blunderbuss.go

```diff
@@ -82,6 +119,30 @@ func printChance(owners weightMap, total int64) {
 	}
 }
 
+// expandAliases takes aliases and expands them into owner lists.
+// It also handles recursive aliases.
+func (b *BlunderbussMunger) expandAliases(toExpand sets.String) sets.String {
+	finalExpanded := sets.String{}
+
+	for {
+		expanded := sets.String{}
+		for _, owner := range toExpand.List() {
+			if val, ok := b.aliasMap[owner]; ok {
```

Comment:
For better encapsulation, lets wrap access to the map in a function of b: `b.ResolveAlias(owner)`